virus/Narnavirus_eurasiamusculus.md

```diff
@@ -0,0 +1,98 @@
+# Characterization of a Ewe-nique virus that only infects skeletal muscle and cardiac tissue
+
+written by: Kevin Yuan
+
+### Narnavirus eurasiamusculus
+
+This virus is a narnavirus that seems to only infect cardiac and skeletal muscle tissue. Additionally, it has only been found in Europe and Asia.
+
+## Abstract
+
+The field of virus discovery is blooming at an unprecedented rate with the rate of new viruses being discovered. Recently, Edgar et al. published a pipeline for rapid virus discovery through analysis of the sequencing read archive [Edgar et al., 2022]. Through this procedure, an unknown virus was discovered in sequencing run SRR13008222. This study sets out to characterize its features. Here we show that this new virus, dubbed Narnavirus eurasiamusculus, is an RNA virus that only infects the skeletal and cardiac muscle tissue of goats and sheep stemming from Europe and Asia. This is a new narnavirus that shares similar traits to other viruses of the same family. Thereby, our study presents another example of the use of this pipeline to find new virus candidates and characterize them. It also provides another example of the diversity of viruses and how little we actually know about them.
+
+## Results
+
+![](img/Narnavirus_eurasiamusculus/figure.PNG)
```

Comment:
This file is missing from the PR

Reply:
Sorry about that. I think I added it into the respective folder now.